Unable to install v 2.1.5 on OSX 10.11.2

Hello, since I want to use `from-uc` I tried to install the last version, however when I try with

`sudo pip install biom-format`

I get
`Command "/usr/local/opt/python/bin/python2.7 -u -c "import setuptools, tokenize;__file__='/private/tmp/pip-build-Ik0tHF/biom-format/setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record /tmp/pip-Pt8mPM-record/install-record.txt --single-version-externally-managed --compile" failed with error code 1 in /private/tmp/pip-build-Ik0tHF/biom-format/`

How can I solve it?
Thanks!

P.S.: This is the complete message:

Collecting biom-format
  Downloading biom-format-2.1.5.tar.gz (307kB)
    100% |████████████████████████████████| 317kB 2.7MB/s 
Requirement already satisfied (use --upgrade to upgrade): click in /usr/local/lib/python2.7/site-packages (from biom-format)
Requirement already satisfied (use --upgrade to upgrade): numpy>=1.3.0 in /usr/local/lib/python2.7/site-packages (from biom-format)
Requirement already satisfied (use --upgrade to upgrade): future>=0.14.3 in /usr/local/lib/python2.7/site-packages (from biom-format)
Requirement already satisfied (use --upgrade to upgrade): scipy>=0.13.0 in /usr/local/lib/python2.7/site-packages (from biom-format)
Requirement already satisfied (use --upgrade to upgrade): pyqi in /usr/local/lib/python2.7/site-packages (from biom-format)
Installing collected packages: biom-format
  Running setup.py install for biom-format ... error
    Complete output from command /usr/local/opt/python/bin/python2.7 -u -c "import setuptools, tokenize;__file__='/private/tmp/pip-build-Ik0tHF/biom-format/setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record /tmp/pip-Pt8mPM-record/install-record.txt --single-version-externally-managed --compile:
    running install
    running build
    running build_py
    creating build
    creating build/lib.macosx-10.11-x86_64-2.7
    creating build/lib.macosx-10.11-x86_64-2.7/biom
    copying biom/__init__.py -> build/lib.macosx-10.11-x86_64-2.7/biom
    copying biom/err.py -> build/lib.macosx-10.11-x86_64-2.7/biom
    copying biom/exception.py -> build/lib.macosx-10.11-x86_64-2.7/biom
    copying biom/parse.py -> build/lib.macosx-10.11-x86_64-2.7/biom
    copying biom/table.py -> build/lib.macosx-10.11-x86_64-2.7/biom
    copying biom/util.py -> build/lib.macosx-10.11-x86_64-2.7/biom
    creating build/lib.macosx-10.11-x86_64-2.7/tests
    copying tests/__init__.py -> build/lib.macosx-10.11-x86_64-2.7/tests
    copying tests/test_err.py -> build/lib.macosx-10.11-x86_64-2.7/tests
    copying tests/test_parse.py -> build/lib.macosx-10.11-x86_64-2.7/tests
    copying tests/test_table.py -> build/lib.macosx-10.11-x86_64-2.7/tests
    copying tests/test_util.py -> build/lib.macosx-10.11-x86_64-2.7/tests
    creating build/lib.macosx-10.11-x86_64-2.7/biom/cli
    copying biom/cli/__init__.py -> build/lib.macosx-10.11-x86_64-2.7/biom/cli
    copying biom/cli/installation_informer.py -> build/lib.macosx-10.11-x86_64-2.7/biom/cli
    copying biom/cli/metadata_adder.py -> build/lib.macosx-10.11-x86_64-2.7/biom/cli
    copying biom/cli/table_converter.py -> build/lib.macosx-10.11-x86_64-2.7/biom/cli
    copying biom/cli/table_head.py -> build/lib.macosx-10.11-x86_64-2.7/biom/cli
    copying biom/cli/table_normalizer.py -> build/lib.macosx-10.11-x86_64-2.7/biom/cli
    copying biom/cli/table_subsetter.py -> build/lib.macosx-10.11-x86_64-2.7/biom/cli
    copying biom/cli/table_summarizer.py -> build/lib.macosx-10.11-x86_64-2.7/biom/cli
    copying biom/cli/table_validator.py -> build/lib.macosx-10.11-x86_64-2.7/biom/cli
    copying biom/cli/uc_processor.py -> build/lib.macosx-10.11-x86_64-2.7/biom/cli
    copying biom/cli/util.py -> build/lib.macosx-10.11-x86_64-2.7/biom/cli
    creating build/lib.macosx-10.11-x86_64-2.7/tests/test_cli
    copying tests/test_cli/__init__.py -> build/lib.macosx-10.11-x86_64-2.7/tests/test_cli
    copying tests/test_cli/test_add_metadata.py -> build/lib.macosx-10.11-x86_64-2.7/tests/test_cli
    copying tests/test_cli/test_show_install_info.py -> build/lib.macosx-10.11-x86_64-2.7/tests/test_cli
    copying tests/test_cli/test_subset_table.py -> build/lib.macosx-10.11-x86_64-2.7/tests/test_cli
    copying tests/test_cli/test_summarize_table.py -> build/lib.macosx-10.11-x86_64-2.7/tests/test_cli
    copying tests/test_cli/test_table_converter.py -> build/lib.macosx-10.11-x86_64-2.7/tests/test_cli
    copying tests/test_cli/test_table_normalizer.py -> build/lib.macosx-10.11-x86_64-2.7/tests/test_cli
    copying tests/test_cli/test_uc_processor.py -> build/lib.macosx-10.11-x86_64-2.7/tests/test_cli
    copying tests/test_cli/test_validate_table.py -> build/lib.macosx-10.11-x86_64-2.7/tests/test_cli
    running egg_info
    writing requirements to biom_format.egg-info/requires.txt
    writing biom_format.egg-info/PKG-INFO
    writing top-level names to biom_format.egg-info/top_level.txt
    writing dependency_links to biom_format.egg-info/dependency_links.txt
    writing entry points to biom_format.egg-info/entry_points.txt
    warning: manifest_maker: standard file '-c' not found
    reading manifest file 'biom_format.egg-info/SOURCES.txt'
    reading manifest template 'MANIFEST.in'
    no previously-included directories found matching 'docs/_build'
    warning: no previously-included files matching '*.pyc' found anywhere in distribution
    warning: no previously-included files matching '*.pyo' found anywhere in distribution
    warning: no previously-included files matching '.git' found anywhere in distribution
    writing manifest file 'biom_format.egg-info/SOURCES.txt'
    copying biom/_filter.c -> build/lib.macosx-10.11-x86_64-2.7/biom
    copying biom/_filter.pyx -> build/lib.macosx-10.11-x86_64-2.7/biom
    copying biom/_subsample.c -> build/lib.macosx-10.11-x86_64-2.7/biom
    copying biom/_subsample.pyx -> build/lib.macosx-10.11-x86_64-2.7/biom
    copying biom/_transform.c -> build/lib.macosx-10.11-x86_64-2.7/biom
    copying biom/_transform.pyx -> build/lib.macosx-10.11-x86_64-2.7/biom
    running build_ext
    building 'biom._filter' extension
    creating build/temp.macosx-10.11-x86_64-2.7
    creating build/temp.macosx-10.11-x86_64-2.7/biom
    clang -fno-strict-aliasing -fno-common -dynamic -g -O2 -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I/usr/local/include -I/usr/local/opt/openssl/include -I/usr/local/opt/sqlite/include -I/usr/local/Cellar/python/2.7.12/Frameworks/Python.framework/Versions/2.7/include/python2.7 -c biom/_filter.c -o build/temp.macosx-10.11-x86_64-2.7/biom/_filter.o
    biom/_filter.c:258:10: fatal error: 'numpy/arrayobject.h' file not found
    #include "numpy/arrayobject.h"
             ^
    1 error generated.
    error: command 'clang' failed with exit status 1
    
    ----------------------------------------
Command "/usr/local/opt/python/bin/python2.7 -u -c "import setuptools, tokenize;__file__='/private/tmp/pip-build-Ik0tHF/biom-format/setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record /tmp/pip-Pt8mPM-record/install-record.txt --single-version-externally-managed --compile" failed with error code 1 in /private/tmp/pip-build-Ik0tHF/biom-format/

--- a/setup.py
+++ b/setup.py
@@ -69,11 +69,14 @@ classifiers = [s.strip() for s in classes.split('\n') if s]
 USE_CYTHON = os.environ.get('USE_CYTHON', False)
 ext = '.pyx' if USE_CYTHON else '.c'
 extensions = [Extension("biom._filter",
-                        ["biom/_filter" + ext]),
+                        ["biom/_filter" + ext],
+                        include_dirs=[np.get_include()]),
               Extension("biom._transform",
-                        ["biom/_transform" + ext]),
+                        ["biom/_transform" + ext],
+                        include_dirs=[np.get_include()]),
               Extension("biom._subsample",
-                        ["biom/_subsample" + ext])]
+                        ["biom/_subsample" + ext],
+                        include_dirs=[np.get_include()])]
 
 if USE_CYTHON:
     from Cython.Build import cythonize

